libs/wire-api/src/Wire/API/Provider/Service.hs

```diff
@@ -300,6 +333,7 @@ data ServiceProfilePage = ServiceProfilePage
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform ServiceProfilePage)
+  deriving (S.ToSchema) via (Schema ServiceProfilePage)
 
 instance ToJSON ServiceProfilePage where
```

Comment:
Aren't these instances redundant now? We could just have them in terms of Schema, right?

Reply:
I am bot aware.